VuFind.js

```diff
@@ -0,0 +1,1804 @@
+		"items": [
+			{
+				"itemType": "book",
+				"title": "A DEPENDENCY TREE ARC FILTER",
+				"creators": [
+					{
+						"firstName": "RENATO SAYAO CRYSTALLINO DA",
+						"lastName": "ROCHA",
+						"creatorType": "author"
+					}
+				],
+				"date": "2018",
+				"abstractNote": "A tarefa de Processamento de Linguagem Natural consiste em analisar linguagens naturais de forma computacional, facilitando o desenvolvimento de programas capazes de utilizar dados falados ou escritos. Uma das tarefas mais importantes deste campo é a Análise de Dependência. Tal tarefa consiste em analisar a estrutura gramatical de frases visando extrair aprender dados sobre suas relações de dependência. Em uma sentença, essas relações se apresentam em formato de árvore, onde todas as palavras são interdependentes. Devido ao seu uso em uma grande variedade de aplicações como Tradução Automática e Identificação de Papéis Semânticos, diversas pesquisas com diferentes abordagens são feitas nessa área visando melhorar a acurácia das árvores previstas. Uma das abordagens em questão consiste em encarar o problema como uma tarefa de classificação de tokens e dividi-la em três classificadores diferentes, um para cada sub-tarefa, e depois juntar seus resultados de forma incremental. As sub-tarefas consistem em classificar, para cada par de palavras que possuam relação paidependente, a classe gramatical do pai, a posição relativa entre os dois e a distância relativa entre as palavras. Porém, observando pesquisas anteriores nessa abordagem, notamos que o gargalo está na terceira sub-tarefa, a predição da distância entre os tokens. Redes Neurais Recorrentes são modelos que nos permitem trabalhar utilizando sequências de vetores, tornando viáveis problemas de classificação onde tanto a entrada quanto a saída do problema são sequenciais, fazendo delas uma escolha natural para o problema. Esse trabalho utiliza-se de Redes Neurais Recorrentes, em específico Long Short-Term Memory, para realizar a tarefa de predição da distância entre palavras que possuam relações de dependência como um problema de classificação sequence-to-sequence. Para sua avaliação empírica, este trabalho segue a linha de pesquisas anteriores e utiliza os dados do corpus em português disponibilizado pela Conference on Computational Natural Language Learning 2006 Shared Task. O modelo resultante alcança 95.27 por cento de precisão, resultado que é melhor do que o obtido por pesquisas feitas anteriormente para o modelo incremental.",
+				"language": "por",
+				"libraryCatalog": "bdtd.ibict.br",
+				"url": "https://www.maxwell.vrac.puc-rio.br/colecao.php?strSecao=resultado&nrSeq=35858@1",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://bdtd.ibict.br/vufind/Search/Results?lookfor=trees&type=AllFields&limit=20&sort=relevance",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://ixtheo.de/Record/1698984863",
```

Comment:
This item does have a DOI -- any way we can capture that from the metadata? (it's in the RIS and I also get it when importing the MARC directly)